tpot/driver.py

```diff
@@ -223,11 +227,10 @@ def main():
 
     tpot = tpot_type(generations=args.GENERATIONS, population_size=args.POPULATION_SIZE,
                      offspring_size=args.OFFSPRING_SIZE, mutation_rate=args.MUTATION_RATE, crossover_rate=args.CROSSOVER_RATE,
-                     cv=args.NUM_CV_FOLDS, n_jobs=args.NUM_JOBS,
-                     scoring=args.SCORING_FN,
+                     cv=args.NUM_CV_FOLDS, n_jobs=args.NUM_JOBS, scoring=args.SCORING_FN,
                      max_time_mins=args.MAX_TIME_MINS, max_eval_time_mins=args.MAX_EVAL_MINS,
                      random_state=args.RANDOM_STATE, config_dict=operator_dict,
-                     verbosity=args.VERBOSITY, disable_update_check=args.DISABLE_UPDATE_CHECK)
+                     verbosity=args.VERBOSITY, dict_lite=args.LITE, disable_update_check=args.DISABLE_UPDATE_CHECK)
```

Comment:
Let's start calling "TPOT lite" as "TPOT light."

TPOT light should be a string parameter to `config_dict`, e.g.,

```TPOTClassifier(..., config_dict='TPOT light')```

And TPOT parses the string 'TPOT light' to using that dictionary.